src/index.js

```diff
@@ -116,13 +115,17 @@ class Masq {
     window.sessionStorage.removeItem('dataEncryptionKey')
   }
 
-  _storeSessionInfo (stayConnected, userId, dataEncryptionKey) {
+  _storeSessionInfo (stayConnected, userId, dataEncryptionKey, username, profileImage) {
     if (stayConnected) {
       window.localStorage.setItem('userId', userId)
       window.localStorage.setItem('dataEncryptionKey', dataEncryptionKey)
+      window.localStorage.setItem('username', username)
+      window.localStorage.setItem('profileImage', profileImage)
     }
     window.sessionStorage.setItem('userId', userId)
     window.sessionStorage.setItem('dataEncryptionKey', dataEncryptionKey)
+    window.sessionStorage.setItem('username', username)
```

Comment:
What do you think about storing an object which contains these 4 properties ? It can be called `currentUser` for instance. It will be eaiser to manage the deletion or check of object existence this way

Reply:
ok done